packages/fether-react/src/Scanner/Scanner.js

```diff
@@ -77,7 +77,15 @@ export default class Scanner extends React.PureComponent {
         {isLoading ? (
           <img alt='loading' src={loading} />
         ) : webcamError ? (
-          <p>{webcamError}</p>
+          <div>
+            <p>
+              {webcamError}&nbsp;Try the&nbsp;
```

Comment:
I'd go for "Visit Fether FAQ to help troubleshooting this issue" with a direct link to the question: https://wiki.parity.io/Fether-FAQ#how-to-fix-a-webcam-error

Reply:
I've pushed a commit addressing your comment